pyproject.toml

```
@@ -1,3 +1,56 @@
+[project]
+name = "SPRAS"
+version = "v0.0.1"
+description = "Signaling Pathway Reconstruction Analysis Streamliner (SPRAS)"
+authors = [
+  { name = "Anthony Gitter", email = "agitter@morgridge.org" },
+]
+license = { file = "LICENSE" }
+readme = "README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
```

```suggestion
    "Operating System :: OS Independent",
    "Development Status :: 3 - Alpha",
    "Intended Audience :: Science/Research",
    "Topic :: Scientific/Engineering :: Bio-Informatics",
```
Pulling from https://pypi.org/classifiers/. Should we sort these?